CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 ## 4.1.3 (Unreleased)
 
+- Fix accepting mention in url and email adresses
```

Comment:
Please mention the ticket number #1327 in the changelog and make sure your entry is paced in the right order according to changelog number.